examples/elevation/components/elevation/elevation.tsx

```diff
@@ -0,0 +1,61 @@
+import {useEffect, useState} from 'react';
+import {
+  useElevationService,
+  useGoogleMap
+} from '@ubilabs/google-maps-react-hooks';
+
+const Elevation = () => {
+  const {map} = useGoogleMap();
+  // Get the elevator from the useElevationService hook
+  const elevator = useElevationService();
+
+  const [infoWindow, setInfoWindow] = useState<google.maps.InfoWindow | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (!map) {
+      return;
+    }
+
+    if (!infoWindow) {
+      // Create a new InfoWindow
+      const initialPosition = {lat: 51.08998021141488, lng: 10.627828045134935};
+      const infowindow = new google.maps.InfoWindow({
+        content: 'Click somewhere on the map to see the elevation',
+        position: initialPosition
+      });
+
+      setInfoWindow(infowindow);
```

Comment:
It's a little bit confusing, that there's an `infoWindow` and an `infowindow` variable.
```suggestion
      const createdInfoWindow = new google.maps.InfoWindow({
        content: 'Click somewhere on the map to see the elevation',
        position: initialPosition
      });

      setInfoWindow(createdInfoWindow);
```

Reply:
Are you okay with calling it `newInfoWindow` ?